scenario-specs/7-user-profiles.md

```diff
@@ -11,10 +11,15 @@ TODO: High level description.
 TODO: Explain the motivation behind enabling this scenario.  
 
 ## Examples  
-TODO: Show a few different examples of what this scenario might look like.  
-
-## Supporting Data  
-TODO: Show supporting data to help explain: why this scenario is important, what this scenario is trying to solve, who this scenario is mainly for.  
+### dApp Buidlers: A team of people building a dApp.
+The "dApp Buidlers" team want to decentralize and open up their organization to the public. They have an existing tooling ecosystem (repositories, social, task management, etc), and ~15 active contributors. They'd like to seamlessly transaction to using their DAO in tandem with all of these other tools. They would also like to support 3 different types of fund disbursement methods: bounties, open proposals, and basic income for the core team members.  
+### Jill: Creator of a growing open source project.  
+TODO: Jill is looking to financially sustain future development without sacrificing her project's sovereignty. She also wants to incentivize new contributors... She's also looking for a way to collaborate with larger organizations that want to take a dependency on her project, and want to influence development...  
+### A-Team: A group of artists that met online, and want to collaborate on a show.  
+TODO: The A-Team is a group of 2D animators that met in a subreddit. They have an idea for a new web series that they want to create...  
 
```

Comment:
Just to provide a little context, @JordanEllis6809 and I thought moving this hear would allow us to break these out into governance templates. We can then design schemas that implement each scenario.

Reply:
Could you elaborate on this? I dont understand. Brake what out into governance templates? And what are governance templates? 

I dont see what these examples have to do with user-profiles?

Reply:
Sorry this is a bit confusing. So these scenarios were ones I originally put in scenario 1 (setup & configure), but they were too broad. We still liked the "picture" they were trying to paint, so we stuck them here in the user profile section because they sort of define different user profiles that could exist.

It really isn't that good of a fit either. I can take these out and just add a "scratch" document with this in it until we find a better home.

Reply:
Definitely misleading with the title and scenario definition. But the motivation behind this was that the governance templates would correlate to the roles that could be defined within the organization. Maybe we should put this somewhere else?

Reply:
I dont know where this fits in right now. I think these examples seem more like global examples that do not belong in a specific scenario.